Formula/woboq_codebrowser.rb

```diff
@@ -1,14 +1,8 @@
 class WoboqCodebrowser < Formula
   desc "Generate HTML from C++ Code"
   homepage "https://code.woboq.org/"
-  url "https://github.com/woboq/woboq_codebrowser/archive/2.0.1.tar.gz"
-  sha256 "f2ec67bb49415986a4bbef515024009433756c83e14fa84f5acc40c83eec19e6"
-
-  bottle do
```

Comment:
removal = altering

>i thought the bottle is for the old version and the bot will re-put a new block? what now, shall i re-add it?

Yes, re-add it, please. Take a look at the commit history to see how this works.